src/atomate2/forcefields/utils.py

```diff
@@ -212,7 +370,78 @@ def relax(
             atoms = atoms.atoms
 
         struct = self.ase_adaptor.get_structure(atoms)
-        return {"final_structure": struct, "trajectory": obs}
+        traj = obs.to_pymatgen_trajectory(None)
+        is_force_conv = all(
+            np.linalg.norm(traj.frame_properties[-1]["forces"][i]) < abs(fmax)
+            for i in range(struct.num_sites)
+        )
+        return ForcefieldResult(
+            final_structure=struct, trajectory=traj, is_force_converged=is_force_conv
+        )
+
+
+def ase_calculator(calculator_meta: str | dict, **kwargs: Any) -> Calculator | None:
+    """
+    Create an ASE calculator from a given set of metadata.
+
+    Parameters
+    ----------
+    calculator_meta : str or dict
+        If a str, should be one of `atomate2.forcefields.MLFF`.
+        If a dict, should be decodable by `monty.json.MontyDecoder`.
+        For example, one can also call the CHGNet calculator as follows
+        ```
+            calculator_meta = {
+                "@module": "chgnet.model.dynamics",
+                "@callable": "CHGNetCalculator"
+            }
+        ```
+    args : optional args to pass to a calculator
+    kwargs : optional kwargs to pass to a calculator
+
+    Returns
+    -------
+    ASE .Calculator
+    """
+    calculator = None
+
+    if isinstance(calculator_meta, str) and calculator_meta in [
+        f"{name}" for name in MLFF
+    ]:
+        calculator_name = MLFF(calculator_meta.split("MLFF.")[-1])
+
+        if calculator_name == MLFF.CHGNet:
+            from chgnet.model.dynamics import CHGNetCalculator
+
+            calculator = CHGNetCalculator(**kwargs)
+
+        elif calculator_name == MLFF.M3GNet:
+            import matgl
+            from matgl.ext.ase import PESCalculator
```

Comment:
@esoteric-ephemera: One thing I'm confused about --- should we be setting `stress_weight=1.0 / 160.21766208` as [here](https://github.com/materialsvirtuallab/matgl/blob/ee9e98700dd636ee6c6ce57e4c7c9128b29af78c/src/matgl/ext/ase.py#L222)? The `Relaxer` class has this in `matgl`, but the `PESCalculator` does not. In general, I'm not really sure what to make of this `stress_weight` parameter because [CHGNet also has this](https://github.com/CederGroupHub/chgnet/blob/bae0331896c64e5b04481187d63e3e47d78a2e9f/chgnet/model/dynamics.py#L159).

Maybe @janosh knows.

Reply:
[it's the conversion factor](https://github.com/CederGroupHub/chgnet/blob/bae0331896c64e5b04481187d63e3e47d78a2e9f/chgnet/model/dynamics.py#L174) from `GPa` to `eV/Angstrom^3`. IIRC, it's there to correct for a unit mismatch between MP and ASE. @BowenD-UCB probably knows better. can we expand the doc string to explain why we need to convert?

Reply:
Thanks! @BowenD-UCB: If it's indeed the case that it's a way to convert MP units of GPa back to ASE units of eV/A^3, it'd probably be better to do this in a more intuitive way with a keyword argument of `pressure_in_GPa: bool = False` or something rather than a user-supplied conversion factor. The phrase "stress_weight" is vague (it's not really a weight; it's a conversion factor on the result, right?), and there is really only one right value for this.

Reply:
i think @kenko911 is the right person to ping about [`PESCalculator`](https://github.com/materialsvirtuallab/matgl/blob/ee9e98700dd636ee6c6ce57e4c7c9128b29af78c/src/matgl/ext/ase.py#L122)

Reply:
I feel that an ASE calculator should be written to correctly interface with ASE and provide physical quantities in the units ASE uses. When I refactored the relaxation workflows, I didn't have to change any of the output quantities we test against, so things are at least consistent with their previous implementation

That being said, we need to figure out what the units being used are for the various calculators and ensure the correct kwargs are being passed through

Reply:
In MatGL, the PESCalculator is primarily utilized as a single-point calculator, with stress units set to GPa. However, for Relaxer and MolecularDynamics functionalities within MatGL, we convert the stress unit from GPa to eV/Å³, aligning with the internal unit of ASE. For other applications in ASE, users are required to specify the appropriate stress unit for PESCalculator. We can also add an option to allow users to select their preferred stress units within PESCalculator.

Reply:
Thanks @kenko911! I've updated the default `calculator_kwargs` for the M3GNet and CHGNet relax/static makers to use `stress_weight = ase.units.GPa`